app/Controllers/userController.php

```diff
@@ -305,8 +305,7 @@ public static function createUser(string $new_user_name, ?string $email, string
 		$configPath = '';
 
 		if ($ok) {
-			$languages = Minz_Translate::availableLanguages();
-			if (empty($userConfig['language']) || !in_array($userConfig['language'], $languages, true)) {
+			if (!Minz_Translate::exists(is_string($userConfig['language']) ? $userConfig['language'] : '')) {
```

Comment:
Does the following works? It could be simpler
```suggestion
			if (!Minz_Translate::exists($userConfig['language'] ?? '')) {
```

Reply:
No it doesn't
```
 ------ -------------------------------------------------------------------------------------------
  Line   app/Controllers/userController.php
 ------ -------------------------------------------------------------------------------------------
  308    Parameter #1 $lang of static method Minz_Translate::exists() expects string, mixed given.
         🪪  argument.type
 ------ -------------------------------------------------------------------------------------------
 ```